src/app/login/components/Login.tsx

```diff
@@ -19,21 +19,26 @@ import { Eye, EyeOff, Lock, Mail } from "lucide-react";
 import { formatPhoneNumber } from "@/utils/formatPhoneNumber";
 
 const formSchema = z.object({
-  phoneNumber: z.string()
-  .min(1, "Phone number is required")
-  .max(12, "Phone number must be at most 12 characters long"),
-  password: z.string()
+  phoneNumber: z
+    .string()
+    .min(1, "Phone number is required")
+    .max(12, "Phone number must be at most 12 characters long"),
+  password: z
+    .string()
     .min(8, "Password must be at least 8 characters long")
     .max(50, "Password must be at most 50 characters long")
     .regex(/[A-Z]/, "Password must contain at least one uppercase letter")
     .regex(/[a-z]/, "Password must contain at least one lowercase letter")
     .regex(/[0-9]/, "Password must contain at least one digit")
-    .regex(/[@#$%^&*()_+!~`|{}\[\]:;'"<>,.?/]/, "Password must contain at least one special character"),
+    .regex(
+      /[@#$%^&*()_+!~`|{}\[\]:;'"<>,.?/]/,
+      "Password must contain at least one special character"
+    ),
```

Comment:
@Mubashar1009  just make zod validation according according to sign in form.

Reply:
removed password field according to the requirements